sunpy/coordinates/transformations.py

```diff
@@ -197,6 +199,67 @@ def hpc_to_hpc(heliopcoord, heliopframe):
     return hpc
 
 
+# The Sun's north pole is oriented RA=286.13 deg, dec=63.87 deg in ICRS, and thus HCRS as well
+# (See Archinal et al. 2011,
+#   "Report of the IAU Working Group on Cartographic Coordinates and Rotational Elements: 2009")
+# The orientation of the north pole in ICRS/HCRS is assumed to be constant in time
+_SOLAR_NORTH_POLE_HCRS = UnitSphericalRepresentation(lon=286.13*u.deg, lat=63.87*u.deg)
+
+
+def _make_rotation_matrix(start_representation, end_representation):
+    """
+    Return the matrix for the direct rotation from one representation to a second representation.
+    The representations need not be normalized first.
+    """
+    A = start_representation.to_cartesian()
+    B = end_representation.to_cartesian()
+    rotation_axis = A.cross(B) / (A.norm() * B.norm())
+    rotation_angle = -np.arcsin(rotation_axis.norm()) # negation is required
+
+    # This line works around some input/output quirks of Astropy's rotation_matrix()
+    matrix = np.array(rotation_matrix(rotation_angle, rotation_axis.xyz.value.tolist()))
+    return matrix
+
+
+@frame_transform_graph.transform(DynamicMatrixTransform, HCRS, HeliographicStonyhurst)
+def hcrs_to_hgs(hcrscoord, hgsframe):
+    """
+    Convert from HCRS to Heliographic Stonyhurst.
+    """
+    # De-tilt the Sun's north pole
+    z_axis = CartesianRepresentation(0, 0, 1)
+    detilt_matrix = _make_rotation_matrix(_SOLAR_NORTH_POLE_HCRS, z_axis)
```

Comment:
We don't need to be calculating this during every call of the transform, it's always the same right?